oauthlib/__init__.py

```diff
@@ -15,3 +15,20 @@
 __version__ = '3.0.2-dev'
 
 logging.getLogger('oauthlib').addHandler(NullHandler())
+
+_DEBUG = False
+
+def set_debug(debug_val):
+	"""Set value of debug flag
+	
+    :param debug_val: Value to set. Must be a bool value.
+	"""
+	global _DEBUG
+	_DEBUG = debug_val
+
+def get_debug_flag():
```

Comment:
`set_debug` & `get_debug_flag` should have symmetric names. I'd suggest `set_debug` / `get_debug` ? other suggestions are ok, but should be symmetric.

Reply:
Refactored it.